.github/workflows/ci.yml

```diff
@@ -59,7 +71,7 @@ jobs:
         docs/_build/
         docs/generated/
         .tox/sample_data/
-      cache-key: docs-${{ github.run_id }}
+      cache-key: ${{ github.sha }}
```

Comment:
Testing if just the sha makes this work. 

Then we need to work out a more complex hashing scheme.